.github/workflows/main.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           rm -rf example_docs/_build_legacy
       - name: Build Furo theme
         run: |
-          THEME=_qiskit_furo tox run -e docs
+          THEME=qiskit tox run -e docs
```

Comment:
Why's this tox syntax different from the examples you provide in the contributing guide, which don't use `run`?

Reply:
Tox 4 recommends using `run` for disambiguation. It's not actually necessary with our environments, but I did it in CI a while ago. But with CONTRIBUTING.md, it's annoying to type more characters so I kept it simple.